Entwurf/Zuarbeit Entwurf.tex

```diff
@@ -0,0 +1,81 @@
+\documentclass[a4paper,10pt]{article}
+\usepackage[utf8]{inputenc}
+\usepackage{tocbibind}
+\usepackage{xcolor}
+\usepackage{tikz}
+\usepackage{hyperref}
+\usepackage[ngerman]{babel}
+\usepackage{graphicx}
+\graphicspath{ {./exports/png/} }
+
+
+\begin{document}
+\section{Grobentwurf}
+Der Grobentwurf in Abbildung \ref{fig:Paketdiagramm_Grobentwurf} wurde so gestaltet, dass er nach der 3-Schichtenarchitektur aufgebaut ist. Genauer gesagt wurde innerhalb dieser, dass Model-View-Controller Muster verwendet. 
+Dabei wurde die View und der Controller für die Präsentationsschicht und das Model für die Logikschicht verwendet.Die Datenhaltungsschicht der 3 Schichtenarchitektur entspricht unserem Repository
+
+\subsection{Einzelne Elemente des MVC}
+  \subsubsection{Model} 
+    Die Models erhalten die Domänendaten.Sie wird durch ihre Funktionalitäten verändert. Die Models werden von allen Paketen Start,Student und Admin ben\"otigt. Dies geht aus der Abbildung \ref{fig:Paketdiagramm_Grobentwurf} hervor.
```

Comment:
Mit \inputenc{utf8} sollte es eigentlich nicht nötig sein, Umlaute so zu escapen, oder? Zur Not einfach XeLaTeX verwenden, dann braucht man nicht mal inputenc

Reply:
Das macht mein Latex-Editor automatisch. Da es auch nicht schadet, würde ich es erstmal drin lassen.